docs/pages/faq.mdx

```diff
@@ -164,18 +164,6 @@ information.
 
 (!docs/pages/includes/compatibility.mdx!)
 
-## Which operating systems are supported?
-
-Teleport binaries runs on Linux, macOS, and Windows.
-
-|Operating System| Version                 |
-|----------------|-------------------------|
-| macOS          | 10.13+ (High Sierra)    |
-| Windows        | Windows 10+ (rev. 1607) |
-| Linux/CentOS   | CentOS 7+               |
-| Linux/Ubuntu   | Ubuntu 14.04+           |
-| Linux/Debian   | Debian 8+               |
-
```

Comment:
Here we're removing content without providing a reference to where this information currently lives. At the very least, we should link to it, but I would argue that this information belongs in FAQ, as this information would be relevant in making the decision to even try Teleport.

Also, I noticed that this block lives in `master`, but hasn't been backported. I'd argue that some form should remain and be added to the live docs.

Reply:
I don't disagree, but for context:

This was mistakenly (and recently) added to the FAQ instead of to the existing Operating System Support page we've had for years.

This PR just moves it back to the page we've always used.

This is why you don't see it outside of master.

Reply:
Makes sense. Thanks for explaining!

Reply:
@alexfornuto https://github.com/gravitational/teleport/pull/16820